config/locales/en.yml

```diff
@@ -334,3 +339,8 @@ en:
     landing_login_body_html: "<strong>Already verified?</strong> Log in to see your publisher dashboard."
     landing_login_button: "Log in"
     landing_begin_button: "Get Started"
+  youtube:
+    authenticate_with_youtube: "Authenticate with YouTube"
+    account_not_found: "Account not found"
```

Comment:
This looks to be emitted when a matching Publisher isn't found. Could we change the copy to something like "Couldn't find a matching publisher record"?